src/tests/test_model.py

```diff
@@ -87,19 +87,28 @@ def test_model_vit_fit(datapipe):
 @pytest.mark.parametrize(
     "litmodule,precision",
     [
-        (CLAYModule, "bf16-mixed" if torch.cuda.is_available() else "32-true"),
-        (ViTLitModule, "bf16-mixed"),
+        (CLAYModule, "16-mixed" if torch.cuda.is_available() else "32-true"),
```

Comment:
Is this ok? My GPU does not support `bf16-mixed`, so if it does not matter this makes my life easier when testing locally.

Reply:
Oh yes, should be ok to change this since it's only on the unit test. But we'll still use `bf16-mixed` when training the actual Foundation Model on A10G GPUs.